diff --git a/dist/ember-template-compiler.js b/dist/ember-template-compiler.js
index 01999af0c4c125af3c713318db7eb85fa492f72d..0c89f7cdda05bf9f7531d3d8f6eda67041f512fb 100644
--- a/dist/ember-template-compiler.js
+++ b/dist/ember-template-compiler.js
@@ -8288,9 +8288,10 @@ var define, require;
             */
             toSlice(expected) {
               const chars = this.data.asString();
-              return void 0 !== expected && chars !== expected &&
-              // eslint-disable-next-line no-console
-              console.warn(`unexpectedly found ${JSON.stringify(chars)} when slicing source, but expected ${JSON.stringify(expected)}`), new SourceSlice({
+              // return void 0 !== expected && chars !== expected &&
+              // // eslint-disable-next-line no-console
+              // console.warn(`unexpectedly found ${JSON.stringify(chars)} when slicing source, but expected ${JSON.stringify(expected)}`),
+              return new SourceSlice({
                 loc: this,
                 chars: expected || chars
               });
